infra/token-transfer-client/src/utils/index.js

```diff
@@ -7,7 +7,14 @@ export const getNextOnboardingPage = user => {
     return null
   } else if (!user.termsAgreedAt) {
     return '/terms'
-  } else if (!user.revisedScheduleAgreedAt && !user.revisedScheduleRejected) {
+  } else if (
+    // Already agreed, don't display revised schedule
+    !user.revisedScheduleAgreedAt &&
```

Comment:
We could also have an enum with 3 values: Agreed, Rejected, Abstained.
And a separate agreedAt column with a date.

Probably not worth the extra complexity.